### PR TITLE
Feature/testing coverage

### PR DESCRIPTION
**Pull request by etgrieco**

### CODE REVIEW

File: js/custom-library-card-menu.module.js [hunk 1]
```diff
@@ -13,9 +13,14 @@ angular
   }])
   .component('prmLibraryCardMenuAfter', {
     controller: 'customLibraryCardMenuController',
-    template: '<button ng-repeat="item in customLibraryCardMenuItems" aria-label="{{ translate(item.description) }}" ng-click="goToUrl(translate(item.action))" class="button-with-icon zero-margin md-button md-primoExplore-theme md-ink-ripple" type="button">'+
-                '<prm-icon style="z-index:1" icon-type="svg" svg-icon-set="{{item.icon.set}}" icon-definition="{{item.icon.icon}}"></prm-icon>'+
-                '<span class="customLibraryCardMenuItem">{{ translate(item.name) }}</span>'+
-              '</button>'
+    template: `<button ng-repeat="item in customLibraryCardMenuItems"
```
Comment (barnabyalter) [inline js/custom-library-card-menu.module.js]: @etgrieco More readable, nice

File: package.json [hunk 2]
```diff
@@ -18,5 +19,20 @@
   "bugs": {
     "url": "https://github.com/nyulibraries/primo-explore-custom-library-card-menu/issues"
   },
-  "homepage": "https://github.com/nyulibraries/primo-explore-custom-library-card-menu#readme"
+  "devDependencies": {
+    "@babel/polyfill": "^7.0.0-beta.42",
```
Comment (barnabyalter) [inline package.json]: @etgrieco what is this one any why do we need it?

File: spec/customLibraryCardMenu.component.spec.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    element = $compile("<prm-library-card-menu-after />")(scope);
+    scope.$digest();
+  }));
+
+  describe('template layout', () => {
+
+    it('should render proper number of buttons', () => {
+      const buttons = element.children();
+
+      Array.from(buttons).forEach(btn => {
```
Comment (barnabyalter) [inline spec/customLibraryCardMenu.component.spec.js]: @etgrieco is this how you got away from the `for` loops?

Comment (etgrieco) [reply spec/customLibraryCardMenu.component.spec.js]: yes, but I also had to pollyfill Array.from to ensure it worked in Karma. One more dependency for cleaner code.

**Pull request merged**